desc/plotting.py

```diff
@@ -2394,6 +2396,12 @@ def plot_coils(coils, grid=None, fig=None, return_data=False, **kwargs):
         ValueError,
         f"plot_coils got unexpected keyword argument: {kwargs.keys()}",
     )
+    errorif(
+        not isinstance(coils, _Coil),
+        ValueError,
+        "Expected `coils` to be object of type `_Coil`, instead got type"
```

Comment:
```suggestion
        "Expected `coils` to be of type `_Coil`, instead got type"
```
just a suggestion as everything in python is an object :-)